scipy/stats/_multivariate.py

```diff
@@ -636,6 +638,30 @@ def cdf(self, x, mean=None, cov=1, allow_singular=False, maxpts=None,
         out = self._cdf(x, mean, cov, maxpts, abseps, releps)
         return out
 
+    def _rvs(self, mean, cov_L, size, random_state=None):
+        """
+        Parameters
+        ----------
+        mean : ndarray
+            Mean of the distribution
+        cov_L : ndarray
+            A decomposition such that np.dot(cov_L, cov_L.T)
+            is the covariance matrix.
+        size : integer
+            Number of samples to draw.
```

Comment:
Not just for integer size; needs to support any number of dimensions, e.g.
```
import numpy as np
cov = np.array([[4, 1], [1, 3]])
mean=np.array([1, 2])
rvs = np.random.multivariate_normal(mean=mean, cov = cov, size=(3, 4))
print(rvs.shape)
```
prints
```
(3, 4, 2)
```

Reply:
Right, I guess I simply followed the `rvs` docstrings.

Reply:
No problem. I think it will be an easy fix.